### PR TITLE
Books, Team, Getting Started ,Quickstart, ScrollSpy and Jumbotron

### PR DESCRIPTION
**Pull request by jacmoe**
Different layout for tablet.
Hidden on mobile.

### CODE REVIEW

**Comment by jacmoe** [comment]
Ok, guys:
I am new to Git, so I think in the future I will squash some of the commits into each other before I push :shipit:

**Comment by jacmoe** [comment]
I decided to push additional 8 commits.
Have actually squashed them down to 8 from 18 commits :)

File: views/site/books.php [hunk 1]
```diff
@@ -7,174 +7,258 @@
...
-
-                    <h4>Recommended for intermediate-to-advanced developers.</h4>
-
-                    <p>The book is based around implementing a real world CRM application following many development best practices.
-                    It is assumed that reader is experienced enough in object orientented programming so don't expect OO-basics
-                    explained. There are many references to good overall programming books so even if you're already familiar with
-                    Yii 2.0 it worth at least skimming it.</p>
+        </div> <!-- book -->
+        <div class="col-sm-3 col-md-3 col-lg-3">
+            <div class="[ info-card ]">
```
Comment (samdark) [inline views/site/books.php]: What are `[` and `]`?

File: views/site/books.php [hunk 2]
```diff
@@ -7,174 +7,258 @@
...
-                <div class="col-sm-6 col-md-9">
-                    <h3>Yii Application Developement Cookbook <small class="authors">by Alexander Makarov</small></h3>
-
-                    <h4>Recommended for intermediate-to-advanced developers.</h4>
-
-                    <p>The book is a set of individual independent recipes written by Yii core developer. Each recipe shows how to
-                    do something useful with Yii explainig why it's done, how it's done, why it works and how exactly it works.</p>
+        </div> <!-- book -->
+        <div class="col-sm-3 col-md-3 col-lg-3">
+            <div class="info-card">
```
Comment (samdark) [inline views/site/books.php]: We could extract book template into partial. It's quite repetitive now.

File: views/site/books.php (hunk 1 above)
Comment (jacmoe) [inline views/site/books.php]: They have - or should have - been removed. :)
I will check..

File: views/site/books.php (hunk 2 above)
Comment (jacmoe) [inline views/site/books.php]: That is a good idea.

File: views/site/books.php (hunk 1 above)
Comment (jacmoe) [inline views/site/books.php]: I will remove the brackets from the class names in a later commit - I probably should not add to this PR?

Comment (samdark) [inline views/site/books.php]: OK.

**Comment by jacmoe** [comment]
Thanks.

Sorry about the huge PR - I guess that you guys were not working during the
week-end, but I was :)

2015-11-30 17:08 GMT+01:00 Alexander Makarov <notifications@github.com>:

**Comment by samdark** [comment]
That's OK. Still manageable. Please fix classes with `[` and `]` and thanks for taking care about website!

**Comment by jacmoe** [comment]
The fix for the brackets is in #78 

I opened that one up and will add to it until you merge/whatever :)

**Pull request merged**